tests/unit/targets/gitlab_suggestion_test.py

```diff
@@ -0,0 +1,22 @@
+#!/bin/env python
+# Copyright: Hallux team, 2023
+
+from __future__ import annotations
+
+from pathlib import Path
+
+from unit.common.testing_issue import TestingIssue
+
+from hallux.proposals.simple_proposal import SimpleProposal
+from hallux.targets.gitlab_suggestion import GitlabSuggestion
+
+
+def test_github_suggestion():
+    (api_url, repo_name, PR_ID) = GitlabSuggestion.parse_mr_url("https://gitlab.com/hallux/hallux/-/merge_requests/2")
+    # assert api_url == "https://api.gitlab.com"
```

Comment:
Delete commented code
```suggestion
    # assert api_url == "https://api.gitlab.com"
```